rules/pravidla.md

```diff
@@ -1,5 +1,5 @@
 Robotour - robotika.cz outdoor delivery challenge
-verze 4 (2017-01-31)
+verze 5 (2017-09-20)
```

Comment:
Shouldn't this date be more ... current?

Reply:
well most changes are from September 2017 ... but OK, I may use YYYY-01-31 as last year